semver/SemanticVersionParser.g4

```diff
@@ -0,0 +1,27 @@
+parser grammar SemanticVersionParser;
+
+options { tokenVocab = SemanticVersionLexer; }
+
+patch: value = NUMBER;
+minor: value = NUMBER;
+major: value = NUMBER;
```

Comment:
No need in these rules if use [rule element labels](https://github.com/antlr/antlr4/blob/master/doc/parser-rules.md#rule-element-labels) (see below).

```suggestion
```